ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -65,7 +65,7 @@ objects:
     name: ${JOB_NAME_SAFE}-bootstrap-exporter
     namespace: ${NAMESPACE}
   spec:
-    host: ${JOB_NAME_SAFE}-bootstrap-exporter-${NAMESPACE}.svc.ci.openshift.org
+    host: ${JOB_NAME_SAFE}-bootstrap-exporter-${NAMESPACE}.apps.build01.ci.devcluster.openshift.com
```

Comment:
Are jobs calling these templates bound to build01? Or is it OK for the ignition config to be on build01 even when the job is running on build02?

Reply:
It looks like the latter is not OK, since the test is not explicitly putting the ignition on build01 but is just putting the ignition in the `/srv` directory.

Reply:
If we cannot determine the cluster on which the test is running, then we will need to add `pull-ci-openshift-installer-master-e2e-aws-upi` to https://github.com/openshift/release/blob/941ca83ec8c6832680b73cf4f94eb1c0beaac9f1/core-services/sanitize-prow-jobs/_config.yaml#L3944-L3956.

Reply:
It looks like you are right. There should be a way to determine the build farm, but will keep this in mind if we can't do it.

Reply:
Or we can just use S3 to host this stuff?  Because I don't see us being able to port the local-Route approach to multi-step templates, when we [migrate this template over][1].

[1]: https://docs.ci.openshift.org/docs/how-tos/migrating-template-jobs-to-multistage/

Reply:
I agree that S3 would be better. I assume that there was some reason why we are not using S3, as I would have thought that would have been the initial approach here when these tests were created.

Reply:
Good point. S3 would also mock properly the documented scenario for UPI so it would be my preferred way to do it. I will look at it.

Reply:
Back when this was written there was one CI cluster, and managing a local Route had fewer moving parts.  But "works in today's world with multiple CI clusters" is more important than "fewer moving parts" ;)